pkg/gce-pd-csi-driver/node_test.go

```diff
@@ -381,26 +382,49 @@ func TestNodeStageVolume(t *testing.T) {
 	stagingPath := filepath.Join(tempDir, defaultStagingPath)
 
 	testCases := []struct {
-		name       string
-		req        *csi.NodeStageVolumeRequest
-		expErrCode codes.Code
+		name          string
+		req           *csi.NodeStageVolumeRequest
+		deviceSize    int
+		blockExtSize       int
+		readonlyBit   string
+		expResize     bool
+		expErrCode    codes.Code
 	}{
 		{
-			name: "Valid request",
+			name: "Valid request, no resize bc size",
```

Comment:
nit: I'd avoid shortening test names. Same for the remaining tests.

```suggestion
                        name: "Valid request, no resize because block and filesystem sizes match",
```